server/coordinator/notify_interface.go

```diff
@@ -0,0 +1,6 @@
+package coordinator
+
+// NotifyInterface is the interface for notifying of new snapshots
```

Comment:
this in particular needs to live somewhere that the server/ code can reference it without creating a dependency on the coordinator